example/resnet50_explicit_path.yml

```diff
@@ -1,7 +1,8 @@
-weight: /home/lym/.modelci/ResNet50/pytorch-torchscript/1.zip
+weight: ~/.modelci/ResNet50/pytorch-torchscript/image_classification/1.zip
```

Comment:
Same problem as stated above

Reply:
> This is a breaking MR, you should also test if the new saving dir can be used for model deployment.
> I feel that the auto-deployment module (https://github.com/cap-ntu/ML-Model-CI/blob/master/modelci/hub/deployer/dispatcher.py) will fail.

I have run the following code 

```python
    from torchvision import models
    model = models.resnet50(pretrained=True)
    from modelci.hub.manager import register_model
    from modelci.types.bo import Framework, IOShape, ModelVersion, Engine, Metric, Task
    from modelci.types.trtis_objects import ModelInputFormat
    # set model input and output formats
    inputs = [IOShape([-1, 3, 224, 224], dtype=float, name='INPUT__0', format=ModelInputFormat.FORMAT_NCHW)]
    outputs = [IOShape([-1, 1000], dtype=float, name='probs')]
    # register the model
    register_model(
        model,
        dataset='imagenet',
        metric={Metric.ACC: 0.76},
        task=Task.IMAGE_CLASSIFICATION,
        inputs=inputs,
        outputs=outputs,
        architecture='ResNet50',
        framework=Framework.PYTORCH,
        version=ModelVersion('2')
    )
    model_info = retrieve_model(architecture_name='ResNet50', framework=Framework.PYTORCH, engine=Engine.TORCHSCRIPT)
    # deploy the model to cuda device 0.
    serve(save_path=model_info[0].saved_path, device='cuda:0', name='torchscript-serving', batch_size=16)
```

and can return a running container, I have tested both `parse_path` and  `generate_path` function before.

Reply:
LGTM